elenacaseyroby/YoutubeMusicLib
# Add "Related Videos" and "Suggested Videos" sections

# What

a. Add "Related Videos" and "Suggested Videos" sections.
b. Stores a listen on video start and a listen on video end.
# Why

a. So user can explore music beyond what they are explicitly searching for.
b. More metadata to analyze later.
# Test
1. Go to http://localhost:5000/play
2. Search for artist.
3. Play video.
4. Notice if "Related Videos" section populates.
5. Listen to a video through to end.
6. Check db to see if listens table.  Should be two entries for that video listen, the latter of which should have listened_to_end = 1 for that row.
7. Merge if all looks well!
